web/toDoList.jsp

```diff
@@ -1,160 +1,147 @@
     isListEmpty = toDoListItems.isEmpty();
 %>
-        <%if (isListEmpty) {%>
-    <div id="emptyListMessage" class="listEmptyMessage">Your To Do List is empty</div>
-        <%}%>
+<%if (isListEmpty) {%>
+<div id="emptyListMessage" class="listEmptyMessage">Your To Do List is empty</div>
+<%}%>
 
-    <div class="toDoListContainerOuter">
-        <div id="listContainer" class="toDoListContainerInner">
-            <div id="listBody" class="toDoListStyle">
-                <%
-                    for (ToDoItem item : toDoListItems) {
-                %>
-                <div id="listItemId-<%= item.ID %>" class="toDoListDataStyle">
+<div class="toDoListContainerOuter">
+    <div id="listContainer" class="toDoListContainerInner">
+        <div id="listBody" class="toDoListStyle">
+            <%
+                for (ToDoItem item : toDoListItems) {
+            %>
+            <div id="listItemId-<%= item.ID %>" class="toDoListDataStyle">
                 <span id="item-<%= item.ID %>" class="itemStyle " contenteditable="true"><%= item.message %>
                 </span>
                 <span class="checkboxStyle checkbox-primary" id="checkboxId-<%= item.ID %>"><input class="checkboxSize"
                                                                                                    type="checkbox"/></span>
+                <%if (item.timer.equals("")) {%>
                 <span id="remainderButton-<%= item.ID %>" class="remainderButtonStyle"
-                      value="Remainder"><span class="glyphicon glyphicon-time"></span>
-                    <span id="displayTimeId-<%= item.ID %>" style="display : none"></span>
-                </span>
-                </div>
-                <div id="panel-<%= item.ID %>" class="panelSlide">
-                    <div class="timerStyle ">
-                        <div class="col-sm-5 input-group date datetimepicker">
-                            <input id="dateId-<%= item.ID %>" type="text" class="form-control"/>
-                            <div class="input-group-addon glyphicon glyphicon-calendar"></div>
-                        </div>
-                    </div>
-                    <div class="timerButtons ">
-                        <div id="cancelButton-<%= item.ID %>" class="col-sm-2 btn btn-default">Cancel</div>
-                        <div id="timeSetButton-<%= item.ID %>" class="col-sm-2 btn btn-default">Done</div>
-                    </div>
-                </div>
-                <%
-                        lastMessageId = item.ID;
-                    }
-                %>
+                      value="Remainder"><span class="glyphicon glyphicon-time"></span></span>
+                <% } else {%>
+                <span class="timeLeftMessageStyle" id="displayTimeId-<%= item.ID %>" style="cursor: pointer"></span>
+                <script> getUserTimeData("<%=item.timer %>", <%= item.ID %>);
```

Comment:
putting script in the middle of html tags is a bad practice. please move this to the buttom `$(function () {` block.